tests/app/service/test_rest.py

```diff
@@ -2149,6 +2150,7 @@ def test_update_service_calls_send_notification_as_service_becomes_live(notify_d
         content_type='application/json'
     )
 
+    zd_send_go_live_service_mock.assert_called()
```

Comment:
Could you switch to `assert_called_once_with` + test the case where go live user ID is set and when it's not?